File: enums.cpp
#include <iostream>
using namespace std;

// Define enum for days
enum Day { Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday };

int main() {
    // Create a variable
    Day today = Wednesday;

    // Print its value
    cout << "Day value: " << today << endl;

    return 0;
}

File: enums.rs
#[derive(Debug)]
enum Day {
    Monday,
    Tuesday,
    Wednesday,
    Thursday,
    Friday,
    Saturday,
    Sunday,
}

fn main() {
    // Create a variable
    let today = Day::Wednesday;

    // Print its value
    println!("Today is: {:?}", today);
}